client/build.shared

```diff
@@ -0,0 +1,70 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+apply plugin: 'checkstyle'
+
+repositories {
+  mavenCentral()
+  maven {
+    url "http://oss.sonatype.org/content/groups/public/"
+  }
+}
+
+project.archivesBaseName='litesockets-http-client'
+
+dependencies {
+  testCompile group: 'junit', name: 'junit', version: '4.12'
+
+  compile (
+    "org.threadly:threadly:$threadlyVersion",
+    "org.threadly:litesockets:$litesocketsVersion",
+    "org.threadly:litesockets-http-protocol:0.6.1-SNAPSHOT",
```

Comment:
Can we put the version in a central file?  We can source a central `build.shared` to reduce the duplicated code.